--- a/browser/locales/en-US/browser/preferences/preferences.ftl
+++ b/browser/locales/en-US/browser/preferences/preferences.ftl
@@ -568,7 +568,7 @@
 # "Firefox" should be treated as a brand and kept in English,
 # while "Home" and "(Default)" can be localized.
 home-mode-choice-default =
-    .label = Firefox Home (Default)
+    .label = Homepage (Default)
 
 home-mode-choice-custom =
     .label = Custom URLs…
@@ -597,8 +597,8 @@
 
 ## Home Section - Firefox Home Content Customization
 
-home-prefs-content-header = Firefox Home Content
-home-prefs-content-description = Choose what content you want on your Firefox Home screen.
+home-prefs-content-header = Homepage Content
+home-prefs-content-description = Choose what content you want on your Homepage screen.
 
 home-prefs-search-header =
     .label = Web Search
@@ -649,9 +649,9 @@
 # and non-essential but also not entirely trivial and useless.
 home-prefs-snippets-header =
     .label = Snippets
 
-home-prefs-snippets-description-new = Tips and news from { -vendor-short-name } and { -brand-product-name }
+home-prefs-snippets-description-new = Tips and news from { -vendor-short-name }
 
 home-prefs-sections-rows-option =
     .label =
